tests/system/marathon_auth_common_tests.py

```diff
@@ -8,7 +8,7 @@
 
 from dcos import marathon
 from urllib.parse import urljoin
-
+from fixtures import user_billy
```

Comment:
What is Billy doing here?

Reply:
Being imported. Effectively calling it's decorator function here : https://github.com/mesosphere/marathon/blob/master/tests/system/fixtures/__init__.py#L59 , effectively making the fixture `user_billy` available for consumption, eventually leading on not failing the `test_authorized_non_super_user` test :smile: